src/NuGet.Clients/NuGet.CommandLine/Common/CommandLineConstants.cs

```diff
@@ -9,5 +9,7 @@ internal static class CommandLineConstants
         internal static string NuGetDocsCommandLineReference = "http://docs.nuget.org/docs/reference/command-line-reference";
 
         internal static string PackagesDirectoryName = "packages";
+
+        internal static string MsbuildPathOnMac = @"/Library/Frameworks/Mono.framework/Versions/Current/lib/mono/msbuild/14.1/bin/";
```

Comment:
Instead of updating this when 15.0 is out and then users get access in the next update, you could check for both the paths right now - `/Library/Frameworks/Mono.framework/Versions/Current/lib/mono/msbuild/14.1/bin/` and ``/Library/Frameworks/Mono.framework/Versions/Current/lib/mono/msbuild/15.0/bin/`. Or just look for the highest versioned directory there? This would be useful since we will be replacing 14.1 with 15.0 very soon.